add_menufunc.js

```diff
@@ -401,6 +401,39 @@ function displayProject(div2c, image, title, subt1, subt2, subt3, iml, tech0, te
   a4chk.textContent = 'See Project';
 }
 
+const form = document.querySelector('form');
+
+function validation() {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.